partiql-spi/src/main/kotlin/org/partiql/spi/fn/builtins/FnAnd.kt

```diff
@@ -27,11 +27,14 @@ internal object Fn_AND__BOOL_BOOL__BOOL : Fn {
     override fun invoke(args: Array<Datum>): Datum {
         val lhs = args[0]
         val rhs = args[1]
+        val lhsIsUnknown = lhs.isNull || lhs.isMissing
```

Comment:
I happened to be checking out the other boolean ops (`OR` and `NOT`). Seems like this logic should also be applied to `OR` and `NOT`